data/fields/line_arrangement.json

```
@@ -0,0 +1,17 @@
+{
+    "key": "line_arrangement",
+    "type": "combo",
+    "strings": {
+        "options": {
+            "horizontal": "Horizontal",
+            "semi_horizontal": "Semi Horizontal",
+            "vertical": "Vertical",
+            "semi_vertical": "semi-vertical",
+            "square": "Square",
+            "triangular": "Triangular",
+            "delta": "Delta",
```

As both of these options are triangular shapes, I think we should clarify the difference:

```suggestion
            "triangular": "Triangular (scalene)",
            "delta": "Triangular (isosceles)",
```